### DISCUSSION TITLE
[Can I use OpenTelemetry .NET 6 for production]

### DISCUSSION THREAD

**Discussion by Hieu-Nguyen-1** [open-telemetry/opentelemetry-dotnet#3830] [Q&A] [answered]
Can I use OpenTelemetry .NET 6 (rc version: 1.0.0-rc9.8) for production?

Thanks you!

**Comment by vishweshbankwar** [comment, marked as answer]
@Hieu-Nguyen-1 If you are referring to this package https://www.nuget.org/packages/OpenTelemetry, the answer is yes. The latest stable version is `1.4.0`

**Comment by bsulliva** [comment]
Just to followup, since I have a similar question, for the auto-instrumenting libraries listed below, is it safe to use a release candidate in production, like 1.0.0-rc9.14?

OpenTelemetry.Instrumentation.AspNetCore
OpenTelemetry.Instrumentation.Http
OpenTelemetry.Instrumentation.SqlClient

**Reply by cijothomas** [reply]
You may use in production, but there could be breaking changes, and also the names of the attributes can change as well.
